core/sessions/orm.go

```diff
@@ -52,58 +56,90 @@ func NewORM(db *sqlx.DB, sessionDuration time.Duration, lggr logger.Logger) ORM
 	return &orm{db, sessionDuration, lggr.Named("SessionsORM")}
 }
 
-// FindUser will return the one API user, or an error.
-func (o *orm) FindUser() (User, error) {
-	return o.findUser()
+// FindUser will attempt to return an API user by email.
+func (o *orm) FindUser(email string) (User, error) {
+	return o.findUser(email)
 }
 
-func (o *orm) findUser() (user User, err error) {
-	sql := "SELECT * FROM users ORDER BY created_at desc LIMIT 1"
-	err = o.db.Get(&user, sql)
+// FindUserByAPIToken will attempt to return an API user by email.
```

Comment:
This comment is wrong, but more broadly, I do not think API tokens are used at all and they should be removed.

Reply:
Ah good catch, updated. Removing would definitely simplify things. In a follow up PR after more investigation?

Reply:
I would say remove beforehand instead to simplify this PR.

Reply:
As discussed offline, ticket cut for later 🙂